lib/Backends/CPU/tests/CPUOperatorTest.cpp

```diff
@@ -0,0 +1,146 @@
+#include "tests/unittests/OperatorTest.h"
+
+#include <iostream>
+
+using namespace glow;
+
+IS_TEST_ENABLED(std::string name) {
+  std::set<std::string> blacklist = {
+      "less_float16Cases/0",
+      "less_int64Cases/0",
+      "ResizeNearest_Float/0",
+      "ResizeNearest_Float16/0",
```

Comment:
It would be nice to use gtest's filter syntax:
1) Users of gtest know it already
2) This syntax allows for usage of (a limited form of) regular expressions, which may make the lists like this much shorter. Or do you intentionally want to use fully specified test names here?

And BTW, if we'd use gtest's filter syntax, may be we could just use the `testing::GTEST_FLAG(filter) = backend_tests_filter_string;` trick? We would just need to form the `backend_tests_filter_string` properly. This could solve multiple problems: 
- You could use gtest filter syntax
- gtest would do the parsing of those regular expressions in the filters (even if we don't go with using the filter flag, it would be still nice to reuse gtest's parsing functionality for filters if possible) 
- gtest would skip execution of corresponding tests
- gtest would report skipped tests as skipped, not as passed
- we could have a flag to ignore the backend-specific filters and try to execute all tests to see if there are new tests that are failing/passing now.

Reply:
I'm not crazy about using GTEST_FLAG(filter) internally to the test.  In particular I don't want to figure out how to merge a user-specified `--gtest_filter` with the internal filter.  It's probably not so bad, but I'm not convinced any of the advantages are worth the added complexity.  I don't see any exposed API for filter parsing, either.

Reply:
> In particular I don't want to figure out how to merge a user-specified --gtest_filter with the internal filter. 

Yeah, I see your point. But I'm not sure they have to interact at all. Typically, a user uses --gtest_filter  to run e.g. a specific test, which means the user knows what she does. I'd say user-provided --gtest_filter should just completely override everything else, i.e. `GTEST_FLAG(filter)` should just be ignored in this case.

BTW, Chromium seems to do some crazy things in this direction ;-) They have their own way to parse those filters and eventually merge them with user-provided ones... I don't think we need to go that far.